docs/UserGuide.md

```diff
@@ -26,7 +27,7 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 
    * **`list`** : Lists all contacts.
 
-   * **`add`**`n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : Adds a contact named `John Doe` to the Address Book.
+   * **`add`**`n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01 r/Admin Assistant l/14 s/3000 hw/40` : Adds a contact named `John Doe` to the Address Book.
```

Comment:
Address Book should be changed to HeRon.

Reply:
Update in the latest commit.